source/Server/Server.csproj

```diff
@@ -12,9 +12,9 @@
     <PackageProjectUrl>https://github.com/OctopusDeploy/JiraIssueTracker</PackageProjectUrl>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Octopus.Data" Version="4.1.0" />
+    <PackageReference Include="Octopus.Data" Version="4.1.1-enh-octopusid0001" />
     <PackageReference Include="Octopus.Diagnostics" Version="1.3.0" />
     <PackageReference Include="Octopus.Server.Extensibility" Version="7.0.0" />
-    <PackageReference Include="Octopus.Shared" Version="4.9.0" />
+    <PackageReference Include="Octopus.Time" Version="1.1.6" />
```

Comment:
Where did this one come from?

Reply:
Yeah, we use an IClock, it used to get pulled in because Shared referenced Time